org.civicrm.reqangular/src/common/services/api/option-group.js

```diff
@@ -1,95 +1,82 @@
-define([
-    'common/angular',
-    'common/lodash',
-    'common/modules/apis',
-    'common/services/api'
-], function (angular, _, apis) {
-    'use strict';
-
-    apis.factory('api.optionGroup', ['$log', 'api', function ($log, api) {
-        $log.debug('api.optionGroup');
+/* eslint-env amd */
 
+define([
+  'common/lodash',
+  'common/modules/apis',
+  'common/services/api'
+], function (_, apis) {
+  'use strict';
 
-        /**
-         * Groups option values by option group name
-         *
-         * @param {array} values
-         * @param {array} groupIds
-         * @return {object} an object formed as such:
-         *    {
-         *        group_name_1: [ { value_1 }, { ... }, { value_n } ],
-         *        ...
-         *        group_name_n: [ { value_1 }, { ... }, { value_n } ]
-         *    }
-         */
-        function valuesByGroup(values, groupIds) {
-            var groupedValues = {};
+  apis.factory('api.optionGroup', ['$log', 'api', function ($log, api) {
+    $log.debug('api.optionGroup');
 
-            _.forEach(groupIds, function (id, name) {
-                groupedValues[name] = _.filter(values, function (value) {
-                    return value.option_group_id === id;
-                });
-            });
+    /**
+     * Normalizes the response values to hide implementation details of the API
+     *
+     * @param  {Object} response
+     * @return {Array} a copy of the values with normalized data
+     */
+    function normalizeResponse (response) {
+      return response.values.map(function (optionValue) {
+        var valueCopy = _.clone(optionValue);
 
-            return groupedValues;
-        }
+        valueCopy.option_group_name = valueCopy['option_group_id.name'];
+        delete valueCopy['option_group_id.name'];
 
-        /**
-         * Returns the ids of the option groups with the given names
-         *
-         * @param {string} groupNames
-         * @return {Promise}
-         *   Resolves to an object with key as the group name and the value
-         *   as the group id
-         */
-        function idsOf(groupNames) {
-            return this.sendGET('OptionGroup', 'get', {
-                name: { 'IN': groupNames },
-                return: ['id', 'name']
-            })
-            .then(function (data) {
-                var idsByName = {};
+        return valueCopy;
+      });
+    }
 
-                data.values.forEach(function (value) {
-                    idsByName[value.name] = value.id;
-                });
+    /**
+     * Groups OptionValues by OptionGroup name
+     *
+     * @param {array} optionValues
+     * @return {object} an object formed as such:
+     *  {
+     *    group_name_1: [ { value_1 }, { ... }, { value_n } ],
```

Comment:
`{ value_1 }` looks like ES6 syntax to me, is it `{ value_1: value_1 }` or [value_1 ,... , value_n] ?